From 3b78caf703af1c68faf6787f83a529249fb4f437 Mon Sep 17 00:00:00 2001
From: Nithin Dabilpuram <ndabilpuram@marvell.com>
Date: Sun, 14 May 2023 14:34:37 +0530
Subject: [PATCH 474/955] ci/ipsec_reassembly: update better reference numbers

Update better reference numbers post rework of fast path
for multi-seg and single seg.

Change-Id: Ia47d8714d4f4ef8560fdf17abcca6ef4d092d3b2
Signed-off-by: Nithin Dabilpuram <ndabilpuram@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/103448
Reviewed-by: Rahul Bhansali <rbhansali@marvell.com>
Reviewed-by: Jerin Jacob Kollanukkaran <jerinj@marvell.com>
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Base-Tests: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
---
 .../ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip.inb | 12 ++++++------
 .../cn10k/rclk2500_sclk1100.106xx.ip_ev.inb          | 10 +++++-----
 .../cn10k/rclk2500_sclk1100.106xx.ip_ev_ss.inb       |  8 ++++----
 .../cn10k/rclk2500_sclk1100.106xx.ip_p.inb           |  8 ++++----
 .../cn10k/rclk2500_sclk1100.106xx.ip_p_ss.inb        |  8 ++++----
 5 files changed, 23 insertions(+), 23 deletions(-)

diff --git a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip.inb b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip.inb
index 2a4c5f4a13611..3bf6d4d5e3ef1 100644
--- a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip.inb
+++ b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip.inb
@@ -1,10 +1,10 @@
 aes-cbc_sha1-hmac
-64: 2930000
-380: 3150000
-1410: 3480000
+64: 3640000
+380: 3650000
+1410: 3900000
 <end>
 aes-gcm
-64: 2920000
-380: 3120000
-1410: 3480000
+64: 3620000
+380: 3620000
+1410: 3840000
 <end>
diff --git a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev.inb b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev.inb
index 7970b4c85e5b8..4d426e97f7336 100644
--- a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev.inb
+++ b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev.inb
@@ -1,10 +1,10 @@
 aes-cbc_sha1-hmac
-64: 7720000
-380: 6800000
+64: 9380000
+380: 6850000
 1410: 3900000
 <end>
 aes-gcm
-64: 7720000
-380: 6800000
-1410: 3900000
+64: 9330000
+380: 6830000
+1410: 3840000
 <end>
diff --git a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev_ss.inb b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev_ss.inb
index 99f3f88ff3464..c44e679f44023 100644
--- a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev_ss.inb
+++ b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_ev_ss.inb
@@ -1,10 +1,10 @@
 aes-cbc_sha1-hmac
-64: 10550000
-380: 10500000
+64: 14320000
+380: 11750000
 1410: 4660000
 <end>
 aes-gcm
-64: 10520000
-380: 10500000
+64: 14350000
+380: 11600000
 1410: 4590000
 <end>
diff --git a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p.inb b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p.inb
index d0f53ad149d0c..8a5b49bc9d9dd 100644
--- a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p.inb
+++ b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p.inb
@@ -1,10 +1,10 @@
 aes-cbc_sha1-hmac
-64: 7200000
-380: 6800000
+64: 9280000
+380: 6830000
 1410: 3900000
 <end>
 aes-gcm
-64: 7200000
-380: 6800000
+64: 9460000
+380: 6830000
 1410: 3800000
 <end>
diff --git a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p_ss.inb b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p_ss.inb
index aec5e3882dc4f..b9d1fd0a77200 100644
--- a/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p_ss.inb
+++ b/marvell-ci/test/cnxk-tests/ipsec_reassembly_perf/ref_numbers/cn10k/rclk2500_sclk1100.106xx.ip_p_ss.inb
@@ -1,10 +1,10 @@
 aes-cbc_sha1-hmac
-64: 10200000
-380: 10100000
+64: 14240000
+380: 11740000
 1410: 4660000
 <end>
 aes-gcm
-64: 10200000
-380: 10100000
+64: 14240000
+380: 11590000
 1410: 4590000
 <end>
-- 
2.25.1

